### PR TITLE
Verify Rari deprecation proposal

### PR DESCRIPTION
**Pull request by thomas-waite**
# Verify Rari deprecation proposal

### CODE REVIEW

File: proposals/description/verify_rari_proposal.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      arguments: (addresses) => [addresses.rariTimelock],
+      description: ''
+    },
+    {
+      target: 'rariDAIPoolManagerProxy',
+      values: '0',
+      method: 'setFundRebalancer(address)',
+      arguments: (addresses) => [addresses.rariTimelock],
+      description: ''
+    },
```
Comment (Joeysantoro) [inline proposals/description/verify_rari_proposal.ts:33]: what do each of these actions do

File: proposals/description/verify_rari_proposal.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      arguments: (addresses) => ['USDC', '336235000000'],
+      description: ''
+    },
+    {
+      target: 'rariDAIPoolManagerProxy',
+      values: '0',
+      method: 'withdraw(string,uint256)',
+      arguments: (addresses) => ['DAI', '94218000000000000000000'],
+      description: ''
+    },
```
Comment (Joeysantoro) [inline proposals/description/verify_rari_proposal.ts:61]: are we sure these aren't higher because of the two fee collection actions above?

Comment (sriyantra) [reply proposals/description/verify_rari_proposal.ts:61]: they're correct, I already simulated the calculations

File: proposals/description/verify_rari_proposal.ts (hunk 1 above)
Comment (sriyantra) [reply proposals/description/verify_rari_proposal.ts:33]: have to setFundRebalancer to the timelock address so the timelock can then claim the fuse fees

**Comment by thomas-waite** [comment]
Updated calldata:

```
0x7d5e81e20000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000022000000000000000000000000000000000000000000000000000000000000003c00000000000000000000000000000000000000000000000000000000000000a80000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000066f4856f1bbd1eb09e1c8d9d646f5a3a193da569000000000000000000000000afd2aade64e6ea690173f6de59fc09f5c9190d74000000000000000000000000c6bf8c8a55f77686720e0a88e2fd1feef58ddf4a000000000000000000000000b465baf04c087ce3ed1c266f96ca43f4847d9635000000000000000000000000c6bf8c8a55f77686720e0a88e2fd1feef58ddf4a000000000000000000000000b465baf04c087ce3ed1c266f96ca43f4847d9635000000000000000000000000c6bf8c8a55f77686720e0a88e2fd1feef58ddf4a000000000000000000000000b465baf04c087ce3ed1c266f96ca43f4847d9635000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb480000000000000000000000006b175474e89094c44da98b954eedeac495271d0f0000000000000000000000008ace03fc45139fddba944c6a4082b604041d19fc0000000000000000000000006f6580285a63f1e886548458f427f8695ba1a563000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000018000000000000000000000000000000000000000000000000000000000000001e0000000000000000000000000000000000000000000000000000000000000024000000000000000000000000000000000000000000000000000000000000002a0000000000000000000000000000000000000000000000000000000000000030000000000000000000000000000000000000000000000000000000000000003400000000000000000000000000000000000000000000000000000000000000380000000000000000000000000000000000000000000000000000000000000044000000000000000000000000000000000000000000000000000000000000005000000000000000000000000000000000000000000000000000000000000000580000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000006600000000000000000000000000000000000000000000000000000000000000024a90655810000000000000000000000008ace03fc45139fddba944c6a4082b604041d19fc000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024a90655810000000000000000000000008ace03fc45139fddba944c6a4082b604041d19fc000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024a90655810000000000000000000000008ace03fc45139fddba944c6a4082b604041d19fc000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024a90655810000000000000000000000008ace03fc45139fddba944c6a4082b604041d19fc00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000426741e7d00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000426741e7d00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008430b39a6200000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000004e492af0c00000000000000000000000000000000000000000000000000000000000000004555344430000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008430b39a6200000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000013f3916963a449e8000000000000000000000000000000000000000000000000000000000000000000034441490000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000044a9059cbb0000000000000000000000005ea4a9a7592683bf0bc187d6da706c6c4770976f0000000000000000000000000000000000000000000000000000004e492af0c0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000044a9059cbb0000000000000000000000004d9629e80118082b939e3d59e69c82a2ec08b4d50000000000000000000000000000000000000000000013f3916963a449e800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000244dd18bf50000000000000000000000006f6580285a63f1e886548458f427f8695ba1a56300000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000430c06f700000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002835261726920476f7665726e616e6365204465707265636174696f6e0a0a20205261726920476f7665726e616e6365204465707265636174696f6e0a0a2020546869732070726f706f73616c20696e2077686f6c652077696c6c20646570726563617465207468652052617269205969656c642041676772656761746f7220616e6420526172692054696d656c6f636b2e0a0a54686520526172692074696d656c6f636b2077696c6c20636c61696d2072656d61696e696e67205553444320616e64204441492066656573207468726f756768206120736572696573206f66207478732e0a0a54686520555344432077696c6c2062652073656e7420746f207468652046757365206d756c74697369672028307835654134413961373539323638336246304263313837643644613730366336633437373039373646292c2077686572652069742077696c6c20626520636f6e76657274656420746f2044414920616e642073656e7420746f207468652054726962652052656465656d65722e0a0a5468697320697320746f206d696e696d697a6520746563686e6963616c207269736b2e200a0a54686520636c61696d65642044414920666565732077696c6c2062652073656e64206469726563746c7920746f207468652072656465656d65722e2046696e616c6c792074686520526172692074696d656c6f636b2077696c6c206861766520697473206f776e657273686970207472616e7366657272656420746f207468652044414f54696d656c6f636b4275726e657220636f6e747261637420283078364636353830323835613633663165383836353438343538663432376638363935424131613536332920666f72206465707265636174696f6e2e0a20200000000000000000000000000000000000000000000000000000000000
```

**Pull request closed without merging**